x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -562,6 +571,15 @@ export const PARAMETERS_DEFINITION = {
         format.length ? format.map(({ label }) => label).join('||') : undefined,
       deserializer: (formats: string): ComboBoxOption[] | undefined =>
         formats.split('||').map(format => ({ label: format })),
+      helpText: (
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.formatHelpText"
+          defaultMessage="Custom formats are allowed using {dateSyntax} syntax."
```

Comment:
I wonder how we could change this to use the active voice. How about "Specify custom formats using {dateSyntax} syntax"?

Reply:
For custom formats, use {dateSyntax}.

Is it ok to remove "syntax" at the end?

Reply:
@gchaps I think that might still be needed. Here's a screenshot, WDYT?

<img width="709" alt="Screen Shot 2020-01-08 at 9 45 03 AM" src="https://user-images.githubusercontent.com/5226211/71987332-fed2c680-31fb-11ea-851e-4888c54f32aa.png">

Reply:
Yep, I see why the word "syntax" is needed. Let's use all caps for the date.  How about this:

Use YYYY/MM/DD syntax for custom formats.

Reply:
I believe the upper/lowercase is intentional so I'm going to leave as is.